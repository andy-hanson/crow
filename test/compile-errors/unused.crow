foo<a> void(p nat, _ str)
	_ act void(nat) = x => ()
	_ act void(nat) = _ => ()
	()

.bar<a> void() compare<a>
	()

.r record mut
	.r0 .mut nat

.u union
	u0 nat

.a alias
	nat

.e enum
	e0

.f flags
	f0
