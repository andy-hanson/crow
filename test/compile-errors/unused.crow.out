exit code: 0
stderr:
test/compile-errors/unused.crow 3:5-3:21 Imported name 'sec' is unused
test/compile-errors/unused.crow 5:5-5:24 Imported name 'time-locale' is unused
test/compile-errors/unused.crow 7:5-7:18 Imported name 'ssize_t' is unused
test/compile-errors/unused.crow 9:5-9:24 Imported name 'begin-end' is unused
test/compile-errors/unused.crow 11:5-11:20 Imported module 'process.crow' is unused
test/compile-errors/unused.crow 16:13-16:14 Local 'p' is unused
test/compile-errors/unused.crow 17:23-17:24 Local 'x' is unused
test/compile-errors/unused.crow 19:5-19:6 Local 'y' is unused
test/compile-errors/unused.crow 22:1-23:7 'bar' is unused
test/compile-errors/unused.crow 25:1-26:17 'r' is unused
test/compile-errors/unused.crow 26:5-26:17 Fields of record 'r' are already private by default.
test/compile-errors/unused.crow 28:1-29:11 'u' is unused
test/compile-errors/unused.crow 31:1-32:8 'a' is unused
test/compile-errors/unused.crow 34:1-35:7 'e' is unused
test/compile-errors/unused.crow 37:1-38:7 'f' is unused
test/compile-errors/unused.crow 40:1-41:15 's' is unused
